packages/resourceful-redux/src/utils/get-resources.js

```diff
@@ -7,16 +7,16 @@ export default function(state, resourceName, idsOrList) {
   const resources = resourceSlice.resources;
   let idsList;
 
-  // This conditional handles the situation where `idsOrList` is an ID
-  if (typeof idsOrList === 'string') {
-    const list = resourceSlice.lists[idsOrList];
+  // This conditional handles the situation where `idsOrRequest` is an ID
+  if (typeof idsOrRequest === 'string') {
+    const list = resourceSlice.requests[idsOrRequest];
     if (!list) {
       return [];
     }
 
     idsList = list;
   } else {
-    idsList = idsOrList;
+    idsList = idsOrRequest;
```

Comment:
This file was correct before. `getResources` pulls from lists, _not_ requests. Lists are ordered subsets of resources that are updated as the user CRUDs around the app, so it's very useful to have a handy way to access the resources stored in a list.

A request's list of IDs, on the other hand, is a static list of IDs that reference what the individual CRUD action returned, which is generally less useful. It's mostly just there for bookkeeping (maybe someone could use that info in a plugin, for instance). But most CRUD apps won't use a request's IDs.